logging-android/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id("com.android.library")
+    kotlin("multiplatform")
```

Comment:
Being that this is Android-only artifact, might make more sense to not use the `multiplatform` plugin?

Reply:
I'm not too worried about this. The _existence_ of this module is temporary, once we have hierarchical sources for Android + JVM, this can be brought back into the main `logging` module.